src/validators/isValidLicensePlate.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Validates a portuguese license plate
+ * @param {String} plateNumber
+ */
+
+export default function isValidLicensePlate(plateNumber) {
+  if (typeof plateNumber !== 'string') return false
+  if (
+    !/(?!(([a-zA-Z]{2})-([a-zA-Z]{2})-(\d{2}))|(\d{2})-([a-zA-Z]{2})-([a-zA-Z]{2})|(\d{2})-(\d{2})-(\d{2})|([a-zA-Z]{2})-([a-zA-Z]{2})-([a-zA-Z]{2}))((\d{2})|([a-zA-Z]{2}))-((\d{2})|(\w{2}))-((\d{2})|(\w{2}))/g.test(
+      plateNumber
+    )
+  )
+    return false
```

Comment:
@JoaoFGuiomar We can remove the if's and returns and just return the result like

```js

const reg = /(?!(([a-zA-Z]{2})-([a-zA-Z]{2})-(\d{2}))|(\d{2})-([a-zA-Z]{2})-([a-zA-Z]{2})|(\d{2})-(\d{2})-(\d{2})|([a-zA-Z]{2})-([a-zA-Z]{2})-([a-zA-Z]{2}))((\d{2})|([a-zA-Z]{2}))-((\d{2})|(\w{2}))-((\d{2})|(\w{2}))/g
return reg.test(plateNumber)
```